docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,13 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Automated host user creation](host-user-creation.mdx)
+- [Dynamic labels](../../management/labels.mdx)
```

Comment:
Is the ability to add labels a feature in itself, or is it more about what you can do with the labels? Since this list is already pretty long, I think we can remove this item.

Reply:
Just dynamic labels, regular labels can be added to agentless nodes. I'll update to reflect that.